packages/neuron-wallet/src/services/node.ts

```diff
@@ -145,7 +143,7 @@ class NodeService {
       return false
     } catch (err) {
       logger.info('CKB:\texternal RPC on default uri not detected, starting bundled CKB node.')
-      const isReadyToStart = await this.detectDependency()
+      const isReadyToStart = await redistCheck()
```

Comment:
The function name could be something like `depsCheck` since `isReadyToStart` doesn't have a strong relationship with `redist`

Reply:
how about rename ```isReadyToStart``` to ```redistReady```?